src/content/docs/es/guides/astro-db.mdx

```diff
@@ -335,13 +335,78 @@ Consulta la documentación de [Drizzle `db.batch()`](https://orm.drizzle.team/do
 ## Astro Studio
 </StudioHeading>
 
-Astro DB puede [conectarse a la plataforma Astro Studio](/es/recipes/studio/) para agregar rápidamente una base de datos alojada a tu proyecto. Puedes ver, gestionar y implementar nuevas bases de datos alojadas, todo desde el portal web de Astro Studio.
+El [portal web de Astro Studio](http://studio.astro.build) te permite conectar y gestionar tus bases de datos de Astro DB alojadas de forma remota a través de una interfaz web o mediante [comandos de CLI](/es/reference/cli-reference/#cli-de-astro-studio).
```

Comment:
Just checking on this edit, because in the English the text doesn't change, just the link is removed?

I think the overwrites that first paragraph that should not be deleted, simply have the link removed, and THEN this should be pasted in?

![image](https://github.com/withastro/docs/assets/5098874/7dcd0a3d-9591-4fd2-844e-e8b18f849a91)

Reply:
You are totally correct I think and just accepted your suggestion, I guess we pasted a bit too much during the T&D session 😆